<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtocolPartsWidget</class>
 <widget class="QWidget" name="ProtocolPartsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitterTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QListWidget" name="topListWidget"/>
     <widget class="QListWidget" name="midListWidget"/>
     <widget class="QListWidget" name="botListWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
